src/tsickle.ts

```diff
@@ -919,19 +926,26 @@ class Annotator extends ClosureRewriter {
       // inserts an artificial `const prefix = goog.require` call for the module, and then registers
       // all symbols from this import to be prefixed.
       if (!this.options.untyped) {
-        let symbols: ts.Symbol[] = [];
+        let symbols: {name: string, sym: ts.Symbol}[] = [];
         const typeChecker = this.program.getTypeChecker();
         if (importClause.name) {
           // import a from ...;
-          symbols = [typeChecker.getSymbolAtLocation(importClause.name)];
+          symbols = [{
+            name: getIdentifierText(importClause.name),
+            sym: typeChecker.getSymbolAtLocation(importClause.name)
+          }];
```

Comment:
I've extracted a method, it's useful together with the comment. The code overall is eerily similar between exports and imports, but different enough that it's annoying to fully unify the larger block (export * vs import * vs import Name).